sc/src/main/webapp/WEB-INF/jsp/tiles/views/home/experiment-list-latest.jsp

```diff
@@ -16,19 +16,9 @@
                 <c:forEach items="${latestExperiments}" var="experimentInfo">
                     <li>
                         <div class="media-object stack-for-small">
-                            <div class="media-object-section middle hide-for-small-only">
-                                <c:choose>
-                                    <c:when test="${experimentInfo.experimentType.baseline}">
-                                        <span data-tooltip class="baseline button-rd" title="Baseline experiment">B</span>
-                                    </c:when>
-                                    <c:otherwise>
-                                        <span data-tooltip class="differential button-rd" title="Differential experiment">D</span>
-                                    </c:otherwise>
-                                </c:choose>
-                            </div>
                             <div class="media-object-section middle hide-for-small-only">
                                 <s:eval expression="T(java.text.NumberFormat).getInstance().format(experimentInfo.numberOfAssays)" var="fmtNumberOfAssays" />
-                                <span class="button secondary no-action">${fmtNumberOfAssays}</span>
+                                <span class="button secondary no-action fixed-width-medium">${fmtNumberOfAssays}</span>
```

Comment:
Also, I think one of the other comments about this was to round the number of cells to something like 10k, 100k, etc. There should be a library that does this (she says naively...).

Reply:
We’ll include a code snippet to do this when the time comes. :)